runtime/kusama/src/weights/xcm/pallet_xcm_benchmarks_generic.rs

```diff
@@ -46,141 +47,199 @@ use sp_std::marker::PhantomData;
 /// Weights for `pallet_xcm_benchmarks::generic`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo<T> {
+	// Storage: unknown `0x3a696e747261626c6f636b5f656e74726f7079` (r:1 w:1)
+	// Proof Skipped: unknown `0x3a696e747261626c6f636b5f656e74726f7079` (r:1 w:1)
+	// Storage: Dmp DeliveryFeeFactor (r:1 w:0)
+	// Proof Skipped: Dmp DeliveryFeeFactor (max_values: None, max_size: None, mode: Measured)
 	// Storage: XcmPallet SupportedVersion (r:1 w:0)
+	// Proof Skipped: XcmPallet SupportedVersion (max_values: None, max_size: None, mode: Measured)
 	// Storage: XcmPallet VersionDiscoveryQueue (r:1 w:1)
+	// Proof Skipped: XcmPallet VersionDiscoveryQueue (max_values: Some(1), max_size: None, mode: Measured)
 	// Storage: XcmPallet SafeXcmVersion (r:1 w:0)
-	// Storage: Dmp DownwardMessageQueueHeads (r:1 w:1)
+	// Proof Skipped: XcmPallet SafeXcmVersion (max_values: Some(1), max_size: None, mode: Measured)
 	// Storage: Dmp DownwardMessageQueues (r:1 w:1)
+	// Proof Skipped: Dmp DownwardMessageQueues (max_values: None, max_size: None, mode: Measured)
+	// Storage: Dmp DownwardMessageQueueHeads (r:1 w:1)
+	// Proof Skipped: Dmp DownwardMessageQueueHeads (max_values: None, max_size: None, mode: Measured)
 	pub(crate) fn report_holding() -> Weight {
-		Weight::from_parts(25_878_000 as u64, 0)
-			.saturating_add(T::DbWeight::get().reads(5 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
+		Weight::from_ref_time(32_612_000 as u64)
```

Comment:
What's this all about? `Weight` should be 2D now.

Reply:
Yes the build fails since it is using that deprecated function. Will update the template.